### PR TITLE
Update normalize-css.js

### PR DESCRIPTION
**Pull request by djbuch**
In some recent cases, msg can be a ConsoleMessage object.

### CODE REVIEW

File: src/normalize-css.js [hunk 1]
```diff
@@ -54,7 +54,11 @@ async function normalizeCssLauncher ({ browser, css, debuglog }) {
 
   page.on('console', msg => {
     // pass through log messages
-    debuglog(msg.replace(/^debug: /, ''))
+    if (typeof msg == "object" && msg.hasOwnProperty('text')) {
```
Comment (pocketjoso) [inline src/normalize-css.js]: change to `===` to fix the linting error and then I'll merge!

**Comment by pocketjoso** [comment]
Thanks for the PR, just fix what I commented and I'll merge.

**Comment by djbuch** [comment]
Hello,
I've made the changes

**Comment by pocketjoso** [comment]
Great, thanks again.

Published as `1.1.1` - cc @bezoerb

**Pull request merged**